packages/application/src/submission.tsx

```diff
@@ -176,10 +177,25 @@ export class SubmissionHandler {
           dialogTitle = 'Job submission to ' + runtime_config + ' succeeded';
           dialogBody = (
             <p>
-              Check the status of your run at{' '}
-              <a href={data.url} target="_blank" rel="noopener noreferrer">
-                Run Details
+              Check the status of your experiment at{' '}
```

Comment:
experiment feel like the wrong word here, why was `run` bad? Wouldn't submission be a better replacement since we use that term elsewhere in our messaging whereas we use experiment nowhere else.

Reply:
![image](https://user-images.githubusercontent.com/382917/83104652-07724f00-a06e-11ea-974c-2e1777621287.png)